inference/environments/minimal-ubuntu20.04-py38-cuda11.6.2-gpu-inference/context/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM mcr.microsoft.com/azureml/inference-base-gpu-2004:{{latest-image-tag}}
+
+RUN conda create -n minimal \
+    python=3.8.13 pip=22.1.2
```

Comment:
I think these can be deleted.

Reply:
I tried to build the image locally even after removing them but its showing the same error CondaError: Unable to create prefix directory '/azureml-envs/minimal'.  Check that you have sufficient permissions.

Reply:
And circling back to my original comment: Those two lines can be deleted.